### PR TITLE
mergecontactform

### PR DESCRIPTION
**Pull request by Alexandre-Rosier**
add link form and contact informations

### CODE REVIEW

**Review by Saurelien — APPROVED** [review]
Accept merging of contact

File: src/components/Objectif.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import './Objectif.css';
 function Objectif(props) {
             return (
                 <div className="Objectif">
-                    <h2>Pourquoi ?</h2>
+                    <h10>Pourquoi ?</h10>
```
Comment (lilkev0804) [inline src/components/Objectif.js:7]: h10 what do you mean ?

**Review by Raph-Nokos — APPROVED** [review]
<h10> ?

**Review by lilkev0804 — APPROVED** [review]
check h10

**Pull request merged**